build-a-backend-with-prisma-in-a-typescript-node-project-ca6628d3/notes/07-prisma-create-a-database-record-with-prisma-client.md

```diff
@@ -0,0 +1,13 @@
+# [Create a Database Record with Prisma Client](https://egghead.io/lessons/prisma-create-a-database-record-with-prisma-client)
+
+
+<TimeStamp start="0:54" end="1:00">
+
+To know more about Thunder Client visit this [page](https://www.thunderclient.io/)
```

Comment:
Should probably change it so that people know what they are clicking. `To know more about Thunder Client, visit their [home page](https://www.thunderclient.io/)`. Something along this line.